test/e2e_node/node_container_manager_test.go

```diff
@@ -183,7 +183,7 @@ func runTest(f *framework.Framework) error {
 
 	expectedNAPodCgroup := cm.ParseCgroupfsToCgroupName(currentConfig.CgroupRoot)
 	expectedNAPodCgroup = cm.NewCgroupName(expectedNAPodCgroup, "kubepods")
-	if !cgroupManager.Exists(expectedNAPodCgroup) {
+	if err := cgroupManager.Exists(expectedNAPodCgroup); err != nil {
 		return fmt.Errorf("Expected Node Allocatable Cgroup Does not exist")
```

Comment:
Might want to fix up the wording of the error assertions here and below to say something to the effect of "expected an error but got nil", etc...